plugins/org.eclipse.reddeer.core/src/org/eclipse/reddeer/core/lookup/WidgetLookup.java

```diff
@@ -213,8 +214,8 @@ public Control findParent(){
 	 */
 	public boolean isExtraShellActive() {
 		Shell activeWorkbenchParentShell = null;
-		if(getWorkbenchLookup() != null){
-			activeWorkbenchParentShell = getWorkbenchLookup().getShellForActiveWorkbench();
+		if(getWorkbenchShellLookup() != null){
+			activeWorkbenchParentShell = getWorkbenchShellLookup().getWorkbenchShell();
```

Comment:
That's the reason of this change?

Reply:
do you mean getWorkbenchShell()/getShellForActiveWorkbench() ? IMHO it is a better naming